packages/sw-cli/src/cli/index.js

```diff
@@ -370,14 +388,164 @@ class SWCli {
     });
   }
 
-  /**
-   * This method will generate the file manifest only.
-   * @return {Promise} The promise returned here will be used to exit the
-   * node process cleanly or not.
-   */
-  buildFileManifest() {
-    // TODO: Build File Manifest
-    return Promise.resolve();
+  _saveConfigFile() {
+    return inquirer.prompt([
+      {
+        name: 'saveConfig',
+        message: 'Last Question - Would you like to save these settings to ' +
+          'a config file?',
+        type: 'confirm',
+        default: true,
+      },
+    ])
+    .then((results) => {
+      return results.saveConfig;
+    })
+    .catch((err) => {
+      logHelper.error(
+        errors['unable-to-get-save-config'],
+        err
+      );
+      throw err;
+    });
+  }
+
+  _buildFileManifestFromGlobs(manifestFilePath, globs) {
+    let globbedFiles = [];
+    globs.forEach((globPattern) => {
+      const files = this._getFileManifestDetails(globPattern);
+      globbedFiles = globbedFiles.concat(files);
+    });
+
+    const manifestEntries = this._filterFiles(globbedFiles);
+
+    return this._writeFilemanifest(manifestFilePath, manifestEntries);
+  }
+
+  _writeFilemanifest(manifestFilePath, manifestEntries) {
+    try {
+      mkdirp.sync(path.dirname(manifestFilePath));
+    } catch (err) {
+      logHelper.error(errors['unable-to-make-manifest-directory'], err);
+      return Promise.reject(err);
+    }
+
+    return new Promise((resolve, reject) => {
+      fs.readFile(
+        path.join(__dirname, '../lib/templates/file-manifest.js.tmpl'), 'utf8',
```

Comment:
You should pass in each of the components of the path to `path.join()` so that it has the intended effect:

```js
path.join(__dirname, '..', 'lib', 'templates', 'file-manifest.js.tmpl')
```